StockManager.cc

```diff
@@ -15,11 +15,17 @@ void StockManager::addStock(int numShares, int marketCap, string name, float ope
 }
 
```

Comment:
Can you add one more function:
- Takes in `int numShares` and `string stockName` and returns whether the stock has enough shares to sell. You can call it something like `bool hasSufficentShares(int numShares, string stockName)`.

Reply:
We will need this in the game manager to check before the user is about to buy a stock

Reply:
Sure, I'm guessing you'll produce exceptions using this? Also, update the UML (in case they mark it after project completion)

Reply:
Yeah I will update the UML once project is complete. And yes we will use this to throw an exception